test/dotnet-performance.Tests/project.json

```diff
@@ -0,0 +1,28 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0-rc2-3002439"
```

Comment:
Should this be *? So it picks up the pkg built from the repo?

If not, what is the strategy for keeping this version updated?